railties/CHANGELOG.md

```diff
@@ -20,4 +20,9 @@
 
     *Steve Polito*
 
+*   Remove the option `config.public_file_server.enabled` from the generators
+    for all environments, as the value is the same in all environments.
+
+    *Adrian Hirt*
```

Comment:
I think we can skip the CHANGELOG entry for this change.

Reply:
Sure, I added it for completeness sake, but fine as well 👍